validations/src/tests.rs

```diff
@@ -6675,6 +6676,64 @@ fn test_block_with_drpool_and_utxo_set<F: FnMut(&mut Block) -> bool>(
     Ok(())
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Add a TODO here because the idea is to eventually turn `minimum_difficulty` into a consensus constant, right? Because then we can remove this function.